app/moderator/moderator.js

```diff
@@ -1,3 +1,6 @@
+import {updateWinner} from '../reducers/game';
```

Comment:
if you're using the store reducer, then you'll only be updating the leader's store and nobody else's.  You need to dispatch something to /storeActions so everyone gets it